.gitmodules

```diff
@@ -12,7 +12,7 @@
 	url = https://github.com/gabime/spdlog
 [submodule "ext/fmt"]
 	path = ext/fmt
-	url = https://github.com/fmtlib/fmt.git
+	url = https://github.com/olupton/fmt.git
```

Comment:
Just fyi: for [similar reasons|https://github.com/fmtlib/fmt/pull/1581] in the past, we have created bbp fork: https://github.com/BlueBrain/fmt. If something doesn't merge sooner, we could use the bbp fork.

Reply:
OK. Note that this fix is only needed for NVHPC 22.3, newer versions don't need it.

Reply:
I pushed the workaround to https://github.com/BlueBrain/fmt; there's also a PR to upstream so we might be able to revert this part.